vllm/distributed/eplb/eplb_state.py

```diff
@@ -348,7 +349,7 @@ def rearrange(self,
         time_start = None
         is_main_rank = ep_rank == 0
         if is_main_rank:
-            torch.cuda.synchronize()
+            current_platform.synchronize()
```

Comment:
Here and below. Same can be done with the standard pytorch API available starting from torch 2.6:
```suggestion
            torch.accelerator.synchronize()
```

Are there actual benefits to define similar device abstraction on vLLM level? Using standard pytorch API will help to have a leaner vLLM code base. See https://docs.pytorch.org/docs/stable/generated/torch.accelerator.synchronize.html#torch.accelerator.synchronize